### PR TITLE
Simple refactorizations (v3).

### PR DESCRIPTION
**Pull request by UmanShahzad**
This builds on top of #838. See last 2 commits for new additions.

Same idea -- simple changes, nothing major, just making code a little cleaner in some places, even if it has no user-facing effects.

See comments for more info on changes (not going to repeat on changes from #838.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
     "test:stylelint": "stylelint './src/**/*.scss' --config ./webpack/.stylelintrc",
     "dev-old": "env NODE_PATH='./src' PORT=8000 UV_THREADPOOL_SIZE=100 node ./webpack/webpack-dev-server.js & env NODE_PATH='./src' PORT=8000 node ./bin/server.js",
     "dev": "env NODE_PATH='./src' PORT=8000 UV_THREADPOOL_SIZE=100 node --expose-gc ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config ./webpack/dev.config.js --progress  & env NODE_PATH='./src' PORT=8000 node --expose-gc ./bin/server.js",
-    "start": "NODE_PATH='src' node ./start",
```
Comment (UmanShahzad) [inline package.json]: This doesn't actually work for me. Haven't tested all the others.

File: src/components/Word/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import bindTooltip from 'utils/bindTooltip';
-import { zeroPad } from 'helpers/StringHelpers';
+import zeroPad from 'helpers/StringHelpers';
```
Comment (UmanShahzad) [inline src/components/Word/index.js]: It's the only function in there so make it default.

File: src/containers/Contact/index.js [hunk 3]
```diff
@@ -111,8 +111,8 @@ class Contact extends Component {
   renderSubmitSuccess = () => (
```
Comment (UmanShahzad) [inline src/containers/Contact/index.js]: This and below, just making lines of relatively equal length, easier to edit if need be.

File: src/locale/ar.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ export default {
     local: 'arabic',
     'local.siteLocale': 'اختر لغة الموقع',
     'local.changeLocal': 'اختر اللغة التي تود عرض الموقع بها',
-    'local.navtiveName': 'العربية',
+    'local.nativeName': 'العربية',
```
Comment (UmanShahzad) [inline src/locale/ar.js]: This and below -- surely unintended typos?

File: src/components/Word/index.js (hunk 2 above)
Comment (naveed-ahmad) [inline src/components/Word/index.js]: But intention was to add string helper in this file :)

Comment (UmanShahzad) [inline src/components/Word/index.js]: Ah :)

File: src/locale/ar.js (hunk 4 above)
Comment (naveed-ahmad) [inline src/locale/ar.js]: yep typos

**Comment by UmanShahzad** [comment]
Build may have failed due to the version change? https://github.com/quran/quran.com-frontend/pull/839/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R164

CI gives the error:

```bash
node-pre-gyp ERR! Tried to download(403): https://node-zopfli.s3.amazonaws.com/Release/zopfli-v2.0.2-node-v48-linux-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for node-zopfli@2.0.2 and node@6.3.0 (node-v48 ABI) (falling back to source compile with node-gyp) 
```

**Comment by UmanShahzad** [comment]
Rebased on the changes and review fixes from #838.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32905

**Comment by UmanShahzad** [comment]
Fixed the previous PR, so this should build now, too.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32907

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32910

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32911

**Comment by UmanShahzad** [comment]
Looks like the commits are properly showing now. (p.s. I could have just rebased on master :) )

**Comment by mmahalwy** [comment]
Thank you @UmanShahzad !!

**Comment by UmanShahzad** [comment]
@mmahalwy No problem, brother! I will continue to contribute inshAllah (mostly over weekends, hopefully the next one; I'm learning React & Ruby for your codebase).

**Pull request merged**